ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -175,7 +176,8 @@ - (void)requestCode:(ADAuthorizationCodeCallback)completionBlock
     THROW_ON_NIL_ARGUMENT(completionBlock);
     [self ensureRequest];
     
-    AD_LOG_VERBOSE(_requestParams.correlationId, @"Requesting authorization code for resource: %@", _requestParams.resource);
+    AD_LOG_VERBOSE(_requestParams.correlationId, @"Requesting authorization code");
+    AD_LOG_VERBOSE_PII(_requestParams.correlationId, @"for resource: %@", _requestParams.resource);
```

Comment:
I think we should repeat the log in the PII one as well, e.g. instead of 

"for resource" saying "Requesting authorization code for resource".